### DISCUSSION TITLE
Compile for Windows from Linux

### DISCUSSION THREAD

**Discussion by airtonix** [wailsapp/wails#2632] [Q&A] [answered]
With another popular golang framework: Ebiten, I was able to compile binaries for windows and linux from linux.

Surely the same is possible with Wails? 

- https://freshman.tech/snippets/go/cross-compile-go-programs/

is it enought to just specify: 

```sh
$ GOOS=windows GOARCH=amd64 wails build ...
$ GOOS=darwin GOARCH=amd64 wails build ...
$ GOOS=linux GOARCH=amd64 wails build ...
```

**Comment by misitebao** [comment, marked as answer]
The Windows platform is implemented using pure Go, which means you can cross-compile to Windows from other platforms, but Mac and Linux use cgo, which cannot be used as a target platform for cross-compilation.

**Reply by airtonix** [reply]
cheers
